lib/upload.js

```diff
@@ -615,6 +615,11 @@ class BaseUpload {
         if (status === 423) {
           this._emitHttpError(req, res, 'tus: upload is currently locked; retry later')
           return
+        } else if (inStatusCategory(status, 500)) {
```

Comment:
Let's not use an `else if` here but just an `if`. The `else` should not be necessary.

Reply:
Done.